packages/components-js/tests/vrt/playwright/helpers/playwright-helper.ts

```diff
@@ -148,37 +148,34 @@ export const setupScenario = async (
   }
 
   await page.setViewportSize({ width: viewportWidth, height: 600 });
-  await page.goto(url);
-  await waitForComponentsReady(page);
 
+  const searchParams = new URLSearchParams();
   if (forceComponentTheme) {
-    await waitForForcedComponentTheme(page, forceComponentTheme);
+    searchParams.append('theme', forceComponentTheme);
   }
-
   if (forceDirMode) {
-    await page.evaluate((forceDirMode) => {
-      document.querySelector('html').setAttribute('dir', forceDirMode);
-    }, forceDirMode);
-  }
-
-  if (scalePageFontSize) {
-    // resize before scaling helps load icons at least in select-wrapper
-    await page.setViewportSize({
-      width: viewportWidth,
-      height: await page.evaluate(() => document.body.clientHeight), // TODO: why dynamic based on content here but fixed 600 everywhere else?
-    });
-  }
-
-  if (emulateMediaPrint) {
-    await page.emulateMedia({ media: 'print' });
+    searchParams.append('dir', forceDirMode);
   }
+  const finalUrl = `${url}?${searchParams.toString()}`;
+  await page.goto(finalUrl);
+  await waitForComponentsReady(page);
 
   await page.setViewportSize({
     width: viewportWidth,
     height: await page.evaluate(() => document.body.clientHeight), // TODO: why dynamic based on content here but fixed 600 everywhere else?
   });
 };
 
+const chunksLink = getComponentChunkLinks({ components: [...COMPONENT_CHUNK_NAMES] }).replace(
+  /https:\/\/cdn\.ui\.porsche\.com\/porsche-design-system/g,
+  'http://localhost:3001'
+);
+
+const iconsLink = getIconLinks({ icons: [...ICON_NAMES] }).replace(
```

Comment:
Spreading isn't needed. Also 98% of icons are unused?

Reply:
Since its readonly the types doesn't match if i use it directly. Importing all of the icons doesn't seem to slow down the test, so is this a problem?